docs/S01-fundamentals/M0-intro/index.html

```diff
@@ -74,7 +74,9 @@ <h3>What's the Proposed Solution?</h3>
 
   <h2>What Makes Up a Protocol?</h2>
 
-  <p>In the following modules, we're going to examine the elements engineers have used to build the peer-to-peer blockchain protocol. We'll then see how systems like Bitcoin and Ethereum join these separate elements together to make a wholly new product: blockchain. Last, we'll talk about the development of Bitcoin and how you can start playing with the Ethereum blockchain right from your browser.</p>
+  <p>
+    In the following modules, we're going to examine the elements engineers have used to build the peer-to-peer blockchain protocol. We'll then see how systems like Bitcoin and Ethereum join these separate elements together to make a wholly new product: blockchain. Last, we'll talk about the development of Bitcoin and how you can start playing with the Ethereum blockchain right from your browser.
```

Comment:
Hi! Could you correct this so the <p> is right next to the text? Will merge when you do that!